code/modules/mob/living/carbon/human/species_types/shells.dm

```diff
@@ -0,0 +1,63 @@
+//�*�*� LORE �*�*�
+/*
+   Human cloning is not a perfect process, and with each replica some genetic information is lost or damaged.
+   This is usually not noticeable, but some people in dangerous professions accumulated enough errors to cause
+   extreme decay, which caused a feedback loop of faster death, more frequent cloning, and worse health.
+
+   Eventually, a new treatment came along to revive the human body at its original levels of functionality, through
+   a combination of experimental genetic and bionic treatments. Some of the malformed were rich enough to afford it,
+   others had to loan their future life to huge companies, such as NT, willing to foot the bill in exchange for very
+   experienced workers.
+
+   This treatment has a further cost. The reanimated bodies' genetic structure is so frayed and unstable that cloning
+   is effectively impossible. The shells, as they've been named, are usually extremely cautious and at times paranoid;
+   they've learned a lot of ways to lose their life, and those that are still around got there by avoiding death at all
+   costs.
+
+   Through a combination of bionic enhancements, odd genetics, and spite, shells can endure extreme amounts of pain and
+   recover from the worst wounds on their own as long as they have a single spark of life left in them.
+*/
+
+
+/datum/species/shell
+	name = "Shell"
+	id = "shell"
+	limbs_id = "shell"
+	sexes = FALSE
+	species_traits = list(EYECOLOR,LIPS,NO_UNDERWEAR)
+	blacklisted = TRUE
+	dangerous_existence = TRUE
+	exotic_blood = "shellblood" //blood is mildly poisonous to others
+	var/decay_countdown = 60 //ticks until the body dies beyond repair, destroying the brain
```

Comment:
What scale of ticks are these?

Could you maybe move it closer (or identical) to the figure for defibrillation?

Reply:
Blood should also do genetic damage, because of shell's unstable genes

Reply:
it's been 4 years man

Reply:
he's right though

Reply:
at least put the comments on the bottom, guys, i thought github was sending me ghost notifications